### ISSUE TITLE
Sync Issues when accessing AE2 Inventory

### ISSUE THREAD

**Issue by awilliamson** [Funwayguy/InfiniteInvo#32]
Whilst in the AE2's crafting CPU info panel, server was spamming

```
[04:29:23] [Server thread/ERROR] [infiniteinvo]: Custom Invo Sync Failed! Only found 0 / 27 requested slots
```

After this I went to access the crafting info from a terminal, and the client immediately crashed with the following message, not sure if it's another mod caused by some network issues in general, or whether it's infiniteinvo specific.

This happened to @Xandaros, I speculated it was more player inventory slot synchronisation issues, so I opened the crafting menu ( which caused problems today see #29 ).

```
[04:29:36] [Netty IO #4/ERROR] [FML]: NetworkDispatcher exception
java.io.IOException: Connection reset by peer
	at sun.nio.ch.FileDispatcherImpl.read0(Native Method) ~[?:1.7.0_75]
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:39) ~[?:1.7.0_75]
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223) ~[?:1.7.0_75]
	at sun.nio.ch.IOUtil.read(IOUtil.java:192) ~[?:1.7.0_75]
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:379) ~[?:1.7.0_75]
	at io.netty.buffer.UnpooledUnsafeDirectByteBuf.setBytes(UnpooledUnsafeDirectByteBuf.java:436) ~[UnpooledUnsafeDirectByteBuf.class:?]
	at io.netty.buffer.AbstractByteBuf.writeBytes(AbstractByteBuf.java:870) ~[AbstractByteBuf.class:?]
	at io.netty.channel.socket.nio.NioSocketChannel.doReadBytes(NioSocketChannel.java:208) ~[NioSocketChannel.class:?]
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:87) [AbstractNioByteChannel$NioByteUnsafe.class:?]
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:480) [NioEventLoop.class:?]
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:447) [NioEventLoop.class:?]
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:341) [NioEventLoop.class:?]
	at io.netty.util.concurrent.SingleThreadEventExecutor$2.run(SingleThreadEventExecutor.java:101) [SingleThreadEventExecutor$2.class:?]
	at java.lang.Thread.run(Thread.java:745) [?:1.7.0_75]
[04:29:36] [Server thread/INFO]: RadonGaming lost connection: TranslatableComponent{key='disconnect.genericReason', args=[Internal Exception: java.io.IOException: Connection reset by peer], siblings=[], style=Style{hasParent=false, color=null, bold=null, italic=null, underlined=null, obfuscated=null, clickEvent=null, hoverEvent=null}}
[04:29:36] [Server thread/INFO]: RadonGaming left the game
```

**Comment by Funwayguy** [comment]
I found the cause of the first issue which may have caused the second from excessive syncing (or an infinite sync loop). An update to fix this should hopefully be out later today

**Comment by Funwayguy** [comment]
Just posted the update. Hopefully shouldn't have this issue again

**Comment by awilliamson** [comment]
Crashing is still happening. Unsure if this is still a sync issue. Or if it's related to this issue at all.

```
Details:
-- Affected level --

	Screen name: appeng.client.gui.implementations.GuiCraftingStatus
Details:
-- Affected screen --

	at net.minecraft.client.gui.inventory.GuiContainer.func_73876_c(GuiContainer.java:698)
Stacktrace:
-- Head --

---------------------------------------------------------------------------------------
A detailed walkthrough of the error, its code path and all known details is as follows:


	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at net.minecraft.client.main.Main.main(SourceFile:148)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:887)
	at net.minecraft.client.Minecraft.func_71411_J(Minecraft.java:962)
	at net.minecraft.client.Minecraft.func_71407_l(Minecraft.java:1650)
	at net.minecraft.client.gui.inventory.GuiContainer.func_73876_c(GuiContainer.java:698)
java.lang.NullPointerException: Ticking screen

Description: Ticking screen
Time: 3/19/15 10:08 PM
```

Occurred when clicking crafting status on AE2. Will try other containers shortly and update accordingly.

**Comment by Xandaros** [comment]
> Before Big Reactors control panel also caused the client crash

It didn't, that was about the first resolved issue.

Also, the latest version seems to fix #30, even though it was meant to fix this :D
